classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
```

Comment:
- Please, to make the relationship possible, you have to add an `attr_reader` for students in Classroom and not an `attr_accessor`
- 👉 `attr_reader :students`
- Open this[ link to learn more](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)